audio/audacity/Portfile

```diff
@@ -192,6 +207,18 @@ configure.args-append \
                     CPPFLAGS=-I${prefix}/include \
                     WX_CONFIG=${wxWidgets.wxconfig}
 
+# the embedded portmidi is linux-only as of v2.2.0
+# building against the system portaudio leads to missing symbols due to
+# mismatched symbol visibility which I haven't managed to address.
```

Comment:
Again, out of pure curiosity. Does it mean that Audacity includes its own copy of `portmidi` (which also comes as a standalone port in MacPorts) which fails to work correctly when built against system libraries? But if you compile against `portmidi` from MacPorts it works OK?

Reply:
Yes and no. Until this version I used the portmidi copy shipped with Audacity, probably as an oversight. That embedded copy can no longer be used because it has received modifications that lock it in to ALSA or some other Linux-specific tech. The build against `port:portmidi` works fine (if it didn't I would have had to strip midi support or add a patch that reverts the mentioned commits).

FWIW, the opposite happened with PortAudio: I did use the copy in MacPorts but that no longer works so I have to use the embedded copy.